src/app/nav-bar/nav-bar.component.css

```diff
@@ -0,0 +1,48 @@
+.nav-container {
+    display: flex;
+    margin: 2vh 2.5vw;
+}
+
+.nav-bar {
+    list-style: none;
+    display: flex;
+    flex-wrap: wrap;
+    width: 100%;
+    border-top: 1px solid var(--light-border);
+    border-bottom: 1px solid var(--light-border);
+}
+
+.nav-bar {
+    padding-inline-start: 2vw;
+}
+
+.nav-bar li {
```

Comment:
convert it to sass and instead of writing it on a line, you can wrote it as:
.navbar {
  li {}
  }